css/style.css

```diff
@@ -1,19 +1,219 @@
 
  /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
+ body{
+  width: 100%;
+  margin: 0px;
+  display: block;
+ }
 
-body {
-    font-family: 'Roboto', sans-serif;
```

Comment:
Roboto is a nicer looking font than the browser defaults and it's something that the project initially had imported. I would in general try to be very specific with the fonts that you use, because otherwise the text will look different on each device.